src/components/capture/creation/DetailsForm.tsx

```diff
@@ -76,21 +73,26 @@ function NewCaptureDetails(props: NewCaptureDetailsProps) {
     };
 
     useEffect(() => {
-        if (connectors.length > 0) {
-            setSchema((previous: typeof schema) => {
-                previous.properties.image.oneOf = connectors.map(
-                    (connector) => {
-                        return {
-                            const: connector.links.images,
-                            title: connector.attributes.name,
-                        };
-                    }
-                );
+        if (isSuccess) {
+            if (connectorsData && connectorsData.data.length > 0) {
```

Comment:
Main change is now we don't personally setup the state in the form so have to do some extra checks.